### PR TITLE
fix 240 sec issue

### PR DESCRIPTION
**Pull request by dev-matan-tsuberi**
This PR silences the 240 sec issue by reaching into the underlying truffle contracts and setting `synchronization_timeout` to `0` (no timeout).

It's important to set it also when obtaining a new contract instance via `.new(..)` or `.at(..)` (but not `.deployed(..)`), though no such instances are present (that are used to fire transactions).

### CODE REVIEW

**Comment by dev-matan-tsuberi** [comment]
[Do not send "operation failed" notification after 240 seconds passed.](https://trello.com/c/42pw11QG/251-do-not-send-operation-failed-notification-after-240-seconds-passed)

File: src/actions/arcActions.ts [hunk 1]
```diff
@@ -587,7 +587,7 @@ export function createProposal(daoAvatarAddress: string, title: string, descript
       const schemeParamsHash = await dao.controller.getSchemeParameters(contributionRewardInstance.contract.address, dao.avatar.address);
       const schemeParams = await contributionRewardInstance.contract.parameters(schemeParamsHash);
       const votingMachineAddress = schemeParams[2];
-      const votingMachineInstance = await Arc.GenesisProtocolFactory.at(votingMachineAddress);
```
Comment (tibetsprague) [inline src/actions/arcActions.ts]: we can remove the votingMachineAddress const no?

File: src/index.tsx [hunk 2]
```diff
@@ -10,6 +10,17 @@ import "./assets/styles/global.scss";
 async function renderApp() {
   try {
     await Arc.InitializeArcJs();
+    // Silence 240 sec error
+    Arc.ContractWrappers.AbsoluteVote.contract.constructor.synchronization_timeout = 0;
```
Comment (tibetsprague) [inline src/index.tsx]: I hope this is fixed by the work @dkent600 is doing to change how arc.js handles transactions

File: src/actions/arcActions.ts (hunk 1 above)
Comment (dev-matan-tsuberi) [reply src/actions/arcActions.ts]: done https://github.com/daostack/alchemy/pull/103/commits/e6e5ebf9f63cfa24e4c88bb950ee25afcc9f9e51

File: src/index.tsx (hunk 2 above)
Comment (dkent600) [reply src/index.tsx]: @tibetsprague Without changing the timeout, truffle will still timeout waiting for mined transactions.

**Pull request merged**